templates/createEventsPage.phtml

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1">
+    <title>Academy Events</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+          integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="./css/genericStyles.css">
+</head>
+<body>
+<div class="container">
+    <h1>Events</h1>
+    <button type="button" class="btn btn-success">go back</button>
+    <h2>Add new Event</h2>
+    <form id='CreateEventsForm'>
+
+        <label for='EventCategory'>Event category</label>
+        <select name='eventCategory' id='eventCategory' class='createEvents form-control'>
+            <option value='0'>Select and event category</option>
+            <?php echo \Portal\ViewHelpers\EventCategoryViewHelper::eventCategoryDropdown($data); ?>
+        </select>
+
+        <label for="eventName">Event Name</label>
+        <input id="eventName" name="eventName" class="createEvents form-control" type="text" placeholder="Event Name"
+               data-required='true' data-max='255'>
+
+        <label for='eventDate'>date</label>
+        <input id="EventDate" name="name" class="createEvents form-control" type="date" data-required='true'
+               data-max='255'>
+
+        <label for="eventLocation">location</label>
+        <input id="location" name="location" class="createEvents form-control" type="text" placeholder="Location"
+               data-required='true' data-max='600'>
+
+        <label for='eventStartTime'>Event start time</label>
+        <input id='eventStartTime' name='EventStartTime' class="createEvents form-control" type="time" data-max='20'>
+
+        <label for='eventEndTime'>Event end time</label>
+        <input id='eventEndTime' name='EventEndTime' class="createEvents form-control" type="time" data-max='20'>
```

Comment:
Times don't have data required but shouldn't be optional

Reply:
On It